### ISSUE TITLE
'from' errors as a VariableDeclaration name

### ISSUE THREAD

**Issue by cgewecke** [federicobond/solidity-parser-antlr#5]
Parser errors on this line in Zeppelin's [ERC20.sol](https://github.com/OpenZeppelin/zeppelin-solidity/blob/master/contracts/token/ERC20.sol#L13)
```javascript
function transferFrom(address from, address to, uint256 value);
```
Triggered by the [Identifier rule](https://github.com/federicobond/solidity-parser-antlr/blob/master/Solidity.g4#L266)? 

(I'm installing directly from the repo).

**Comment by federicobond** [comment]
Thanks for the report. I have a fix for this in the works. Will update soon.

**Comment by federicobond** [comment]
Fixed in 47f594ce508532280bbb76b695cd971c30d98868.
